code/datums/components/buildableturf.dm

```diff
@@ -50,6 +50,8 @@ TYPEINFO(/datum/component/buildable_turf)
 						clear_edge_overlays(location)
 						T.vis_contents -= station_repair.ambient_obj
 					return TRUE
+	if(istype(I,/obj/item/magnetizer))
+		return
```

Comment:
d6bdf34a0b6fb71fd49fe8ca80fc6b52971efa7a _should_ resolve everything but this change... what does this resolve?